smsdk/smsdk_entities/machine/machine_v1.py

```diff
@@ -64,3 +65,39 @@ def get_type_from_machine_name(self, machine_source, *args, **kwargs):
             ):
                 machine_type = record["type"]
         return machine_type
+
+    def modify_query_params(self, url, kwargs):
+        where = []
+        order_by = []
+        params = {}
+        for key, value in kwargs.items():
+            where_query = {}
+            orderby_query = {}
+            select_query = {}
+            if not key.startswith("_"):
+                where_query["name"] = key
+                where_query["value"] = value
+                where.append(where_query)
+            elif key == "_order_by":
+                orderby_query["name"] = (
+                    value.replace("-", "") if value.startswith("-") else value
+                )
+                orderby_query["order"] = "desc" if value.startswith("-") else "asc"
+                order_by.append(orderby_query)
```

Comment:
If the key is an unknown option? `_limit`, should the code print a warning? error?  It there is never an invalid key, add a comment stating that behavior. Or maybe a comment we ignore other keywords.

Reply:
Added `_limit` implementation and added a comment stating that we are ignoring other keys apart from the keys exist in get_machine() implementation.